app/src/main/java/com/example/tester/RegisterUser.java

```diff
@@ -31,8 +31,25 @@ protected void onCreate(Bundle savedInstanceState) {
 
             try {
                 registerUserValidation.validateRegister(myDB, username, password, repassword);
+
                 Toast.makeText(RegisterUser.this, "Registration Successful.", Toast.LENGTH_SHORT).show();
```

Comment:
Here, it doesn't seem to be aligned with our current main branch.
Android Toasts are all currently replaced with Prompts, but it seems to be alive in this PR.